File: rest/taskrouter/worker-channels/list/get/example-1/example-1.9.x.java
import com.twilio.Twilio;
import com.twilio.base.ResourceSet;
import com.twilio.rest.taskrouter.v1.workspace.worker.WorkerChannel;

public class ListWorkerChannel {
	private static final String ACCOUNT_SID = System.getenv("TWILIO_ACCOUNT_SID");
	private static final String AUTH_TOKEN = System.getenv("TWILIO_AUTH_TOKEN");
	private static final String WORKSPACE_SID = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
	private static final String WORKER_SID = "WKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";

	public static void main(String[] args) {
		Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

		ResourceSet<WorkerChannel> channels = WorkerChannel.reader(WORKSPACE_SID, WORKER_SID).read();

		for (WorkerChannel wc : channels) {
			System.out.println(wc.getTaskChannelUniqueName());
		}
	}

}

File: rest/taskrouter/worker-reservations/list-get-example1/example-1.9.x.py
# Download the Python helper library from twilio.com/docs/python/install
import os
from twilio.rest import Client

# Your Account Sid and Auth Token from twilio.com/user/account
# To set up environmental variables, see http://twil.io/secure
account_sid = os.environ['TWILIO_ACCOUNT_SID']
auth_token = os.environ['TWILIO_AUTH_TOKEN']
workspace_sid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
worker_sid = "WKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"

client = Client(account_sid, auth_token)

reservations = client.taskrouter.workspaces(workspace_sid) \
    .workers(worker_sid).reservations.list()

for reservation in reservations:
    print(reservation.reservation_status)
    print(reservation.worker_name)
